custom_typings/polymer-build.d.ts

```diff
@@ -76,6 +76,8 @@ declare module 'polymer-build' {
   }
 
   export interface AddServiceWorkerOptions {
+    project: PolymerProject;
```

Comment:
Oh cool, I didn't know about this feature. 

I have something else relying on this that I want to get into PR, so I'm going to merge now but will look into this.